src/api/hooks.ts

```diff
@@ -136,6 +149,7 @@ export const useAPILazyUnsafe = <
       // argsWithCsrf doesn't typecheck as TArgs, so we force the issue.
```

Comment:
Could this be fixed by declaring `const argsWithCsrf: TArgs`?

Reply:
Nope, the assignment won't go through: `Type 'Pick<TArgs, Exclude<keyof TArgs, "csrf">> & { csrf: string; }' is not assignable to type 'TArgs'.
  'Pick<TArgs, Exclude<keyof TArgs, "csrf">> & { csrf: string; }' is assignable to the constraint of type 'TArgs', but 'TArgs' could be instantiated with a different subtype of constraint 'UnsafeRequestArgs'.`

Reply:
That's quite unfortunate.